### PR TITLE
Load assets locally instead of from S3

### PR DESCRIPTION
**Pull request by brianpeiris**
The pen, duck and loading environment models were previously being loaded from a couple of S3 buckets when they didn't need to be. This PR just moves them to local assets.

### CODE REVIEW

File: src/utils/resolveURL.js [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-  return path + url;
-}
-
-export function extractUrlBase(url) {
-  const index = url.lastIndexOf("/");
-
-  if (index === -1) return "./";
-
-  return url.substr(0, index + 1);
-}
```
Comment (brianpeiris) [inline src/utils/resolveURL.js]: This code was unused.

File: src/utils/url-utils.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export function getAbsoluteUrl(baseUrl, relativeUrl) {
+  return new URL(relativeUrl, baseUrl);
+}
+export function getAbsoluteHref(baseUrl, relativeUrl) {
+  return getAbsoluteUrl(baseUrl, relativeUrl).href;
+}
```
Comment (gfodor) [inline src/utils/url-utils.js]: perhaps these should go into `media-url-utils`?

File: src/hub.html [hunk 3]
```diff
@@ -359,7 +361,7 @@
                 <a-entity
                     class="pen interactable"
                     body-helper="type: dynamic; mass: 0.001; collisionFilterGroup: 1; collisionFilterMask: 15; gravity: 0 0 0;"
-                    media-loader="src: https://asset-bundles-prod.reticulum.io/interactables/DrawingPen/DrawingPen-34fb4aee27.gltf;"
+                    gltf-model-plus="src: #drawing-pen;"
```
Comment (gfodor) [inline src/hub.html]: interesting, doesn't this change the behavior a bunch? (no loading sound, cube, spawn animation, etc?) probably for the better if it feels good.

Comment (brianpeiris) [reply src/hub.html]: Yeah, that was my thinking as well, it does feel snappier without the loading and the unnecessary URL resolution.

**Pull request merged**